Main/Controllers/CollectionsController.cs

```diff
@@ -47,5 +47,30 @@ public IActionResult AddCollection([FromBody] CollectionCreationData data)
       _dbContext.SaveChanges();
       return Ok();
     }
+
+    [HttpDelete]
+    public IActionResult DeleteCollection([FromBody] int id)
+    {
+      var userId = _userProvider.GetCurrentUserId();
+      var collection = _dbContext.Collection.Single(c => c.Id == id);
```

Comment:
What is the collection is shared? I don't think those records would be deleted then. The same goes for SavedCollections in the future.